Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -238,14 +238,14 @@ namespace Multiplayer
         if (AZ::Interface<IMultiplayer>::Get()->Connect(editorsv_serveraddr.c_str(), sv_port))
         {
             AZ_Printf("MultiplayerEditorConnection", "Editor-server ready. Editor has successfully connected to the editor-server's network simulation.")
-            AZ::Interface<IMultiplayerEditorConnectionViewportMessage>::Get()->StopViewportDebugMessaging();
+            AZ::Interface<IMultiplayerConnectionViewportMessage>::Get()->StopCenterViewportDebugMessaging();
         }
         else
         {   
             char connection_fail_message[256];
```

Comment:
I believe the preference here is to use AZStd::fixed_string<256>

Reply:
much cleaner :)